src/FMI2/md.jl

```diff
@@ -171,6 +174,71 @@ function getDerivativeIndices(node::EzXML.Node)
     sort!(indices, rev=true)
 end
 
+# helper function to parse variable attributes and set the corresponding property 
+# for element `targetArray[index]`
+function parseModelDescriptionVariable(defnode, _typename=nothing)
+    typename = isnothing(_typename) ? defnode.name : _typename
+    if typename == "Real"
+        typenode = fmi2ModelDescriptionReal()
+        if haskey(defnode, "quantity")
+            typenode.quantity = defnode["quantity"]
+        end
+        if haskey(defnode, "unit")
+            typenode.unit = defnode["unit"]
+        end
+        if haskey(defnode, "displayUnit")
+            typenode.displayUnit = defnode["displayUnit"]
+        end
+        if haskey(defnode, "relativeQuantity")
+            typenode.relativeQuantity = parseBoolean(defnode["relativeQuantity"])
+        end
+        if haskey(defnode, "min")
+            typenode.min = parseReal(defnode["min"])
+        end
+        if haskey(defnode, "max")
+            typenode.max = parseReal(defnode["max"])
+        end
+        if haskey(defnode, "nominal")
+            typenode.nominal = parseReal(defnode["nominal"])
+        end
+        if haskey(defnode, "unbounded")
+            typenode.unbounded = parseBoolean(defnode["unbounded"])
+        end
+        if haskey(defnode, "start")
+            typenode.start = parseReal(defnode["start"])
+        end
+        if haskey(defnode, "derivative")
+            typenode.derivative = parse(UInt, defnode["derivative"])
+        end
+    elseif typename == "String"
+        typenode = fmi2ModelDescriptionString()
+        if haskey(defnode, "start")
+            targetArray[index].String.start = defnode["start"]
```

Comment:
```suggestion
            typenode.start = defnode["start"]
```

Reply:
You are right, this is an historic artifact. Sorry!